bundles/org.openhab.binding.tr064/src/main/resources/OH-INF/i18n/tr064_de.properties

```diff
@@ -51,7 +51,7 @@ thing-type.config.tr064.subdevice.refresh.label = Abfrageintervall
 thing-type.config.tr064.subdevice.uuid.label = UUID
 thing-type.config.tr064.subdevice.uuid.description = UUID des Untergerätes.
 thing-type.config.tr064.subdeviceLan.macOnline.label = MAC-Adresse(n) Online
-thing-type.config.tr064.subdeviceLan.macOnline.description = Komma getrennte Liste mit MAC-Adressen für "Online" Erkennung (Format\: 11\:11\:11\:11\:11\:11\:11\:11\:11).
+thing-type.config.tr064.subdeviceLan.macOnline.description = Liste mit MAC-Adressen für "Online" Erkennung (Format\: 11\:11\:11\:11\:11\:11\:11\:11\:11). Jede Zeile eine MAC-Adresse.
```

Comment:
Please exclude this change from the PR and instead provide it through Crowdin: https://crowdin.com/translate/openhab-addons/1214/en-de?filter=basic&value=0

For reference: https://www.openhab.org/docs/developer/utils/i18n.html#managing-translations

I will remove "Komma getrennte" in Crowdin now, so that at least this part will make it into release 3.4 in case the other part is not fully clear (didn't read through all issue comments).

Reply:
It was actually already fixed a year ago by @cweitkamp. but the wrong version still existed and was the approved one. Approved the correct one and deleted the other.

Reply:
#13990 merged.